src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml

```diff
@@ -89,13 +89,13 @@ limitations under the License.
       <access op="write" role="manage"/>
     </attribute>
 
-    <attribute side="server" code="0x0030" define="PHYSICAL_CONTACT_OCCUPIED_TO_UNOCCUPIED_DELAY" type="int16u" writable="true" default="0x0000" optional="true">
+    <attribute side="server" code="0x0030" define="PHYSICAL_CONTACT_OCCUPIED_TO_UNOCCUPIED_DELAY" type="int16u" writable="true" default="0x0000" optional="true" isNullable="true">
```

Comment:
This is a backwards incompatible API and wire change.  We need to change the spec instead.

Reply:
Looking at the spec, this is also a bit strange. The PhysicalContactOccupiedToUnoccupiedDelay and PhysicalContactUnoccupiedToOccupiedDelay are the only delay attributes which is nullable.

I can make a PR to remove the X quality and remove "The null value indicates that the sensor does not report occupied to unoccupied transition." statement from the attributes? @andreilitvin?

Reply:
I think we should change the spec here: make things consistent and also not break backwards wire compatibility (which we certainly cannot do)

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7799